apps/backend/supabase/functions/.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+  "deno.enable": true,
+  "deno.lint": true,
+  "editor.defaultFormatter": "denoland.vscode-deno"
```

Comment:
question: where/how are we using deno in our codebase?

Reply:
Supabase uses deno for it's functions runtime. We aren't running those, but this just came as part of the set up for a supabase project.